src/backend/marsha/bbb/tests/test_api.py

```diff
@@ -152,10 +150,10 @@ def test_api_fetch_list_instructor(self):
         """An instructor should not be able to fetch a classroom list."""
         classroom = ClassroomFactory()
 
-        jwt_token = AccessToken()
-        jwt_token.payload["resource_id"] = str(classroom.id)
-        jwt_token.payload["roles"] = [random.choice(["instructor", "administrator"])]
-        jwt_token.payload["permissions"] = {"can_update": True}
+        jwt_token = LTIInstructorOrAdminResourceAccessTokenFactory(
+            resource=classroom,
+            permissions__can_update=True,
+        )
```

Comment:
No, it sets `can_access_dashboard` to True, because in the code, `can_update` is true only when the LTI context ID is the resource's playlist LTI ID (and user is admin or instructor).

Reply:
Oh yes, ok